addons/aircraft/functions/fnc_gau8_muzzleEffect.sqf

```diff
@@ -0,0 +1,42 @@
+/*
+ * Author: Alexus, Zabuza, SzwedzikPL, Sargken
+ * GAU-8 muzzle effect
+ *
+ * Argument:
+ * 0: Unit <OBJECT>
+ * 1: Weapon <STRING>
+ * 3: Muzzle <STRING>
+ * 4: Ammo <STRING>
+ * 5: Magazine <STRING>
+ * 6: Projectile <OBJECT>
+ *
+ * Return value:
+ * None
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+private ["_weapon", "_projectile", "_i", "_no"];
+
+_weapon =  param [1];
+_projectile = param [6];
```

Comment:
`params ["", "_weapon", "", "", "", "", "_projectile"];`

Although in this case it's probably better to do:
```
private _weapon = _this select 1;
private _projectile = _this select 6;
```